src/env.rs

```diff
@@ -147,7 +147,7 @@ mod tests {
 
     use self::byteorder::{ByteOrder, LittleEndian};
     use self::tempdir::TempDir;
-    use std::fs;
+    use std::{fs, str};
```

Comment:
In Mentat we standardized on always using multi-line imports, always with a trailing comma, 'cos otherwise six months from now you'll end up reformatting all of these. Your choice whether to follow suit!